deps/wazuh_testing/wazuh_testing/modules/logcollector/event_monitor.py

```diff
@@ -78,6 +83,47 @@ def check_syslog_messages(message, error_message, prefix, file_monitor=None, tim
         timeout (int): Timeout to check the log.
         escape (bool): Flag to escape special characters in the pattern.
     """
-    callback_msg = fr"DEBUG: Reading syslog message: '{message}'"
+    if error_message is None:
+        error_message = f"Did not receive the expected 'Reading syslog message: {message}' event"
+
+    callback_msg = fr".*DEBUG: Reading syslog message: '{message}'.*"
+
     check_logcollector_event(file_monitor=file_monitor, timeout=timeout, callback=callback_msg,
                              error_message=error_message, prefix=prefix, escape=escape)
+
+
+def check_ignore_restrict_messages(message, regex, tag, prefix, error_message=None, file_monitor=None, timeout=T_10,
+                                   escape=False):
+    """Create a callback to detect "DEBUG: Ignoring the log ... due to config" debug line.
+    Args:
```

Comment:
```suggestion

    Args:
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3582/commits/16549a24c6fb60f09d71951f4b9c1c7b68894468